history.md

```diff
@@ -0,0 +1,93 @@
+The original Humanitarian Free and Open Source Software (HFOSS) seminar class was accredited and students in that class commonly got introduced and involved in the student group as well.
+Remy and FOSSBox started the [course-to-coop life cycle](https://opensource.com/education/10/3/course-co-op-lifecycle-openinnovationrit) where students in Open Source would receive co-op offers from companies that were funding and interested in Open Source at RIT.
+This cycle repeated as those students graduated and got full-time offers and referred those who were in the student club for co-op positions.
+FOSSBox may have been an informal student group at first, but it grew to the point where a full-time staff member was managing FOSSBox and undergraduate students had part-time paid roles doing the secretarial work for FOSSBox, both of which were funded by a yearly payment by Red Hat.
+The FOSSBox community operated until approximately 2015, collaborating with organizations in the local area such as [Hacks/Hackers](https://www.hackshackers.com/), [BarCampROC](https://rocwiki.org/BarCamp_Rochester), and the student club [CodeRIT](https://github.com/codeRIT).
+
+
+
+## LibreCorps
+
+Founded in 2012, LibreCorps' first purpose was as a co-op program after developing educational games under the 'One Laptop Per Child' program.
+LibreCorps serves as an experiential learning program where students are deployed to work on real-world Open Source software and data projects, supporting faculty partners with direct software development as well as community management, documentation, design, and other forms of non-code contribution.
+RIT LibreCorps has been partnered intermittently with the [UNICEF Office of Innovation](https://www.unicef.org/innovation/) since 2014.
+As a successful pilot, LibreCorps reached up to 24 supported faculty-driven software projects!
+
+
+
+## FOSS Minor
+
+Beginning of the Fall 2014 Semester, Rochester Institute of Technology started to offer the first minor in [Free and Open Source Software and Free Culture](https://www.rit.edu/study/free-and-open-source-software-and-free-culture-minor).
+The minor was brought about by additional courses offered by the School of Interactive Games and Media ([IGM](https://www.rit.edu/computing/school-interactive-games-and-media)).
+
+
+
+## FOSS@RIT:
+
+FOSS@RIT is the name of the GitHub organization for all things open-source at Rochester Institute of Technology, however, the organization isn't very official. 
+During the period of renaming from FOSSBOX to FOSS@MAGIC, this was what you would call the Open-Source hub, though interchangeable with the FOSS@MAGIC name.
+The FOSS@RIT community was full of people with an interest in free and open-source software at RIT. At the time, FOSS@RIT had the initial purpose of keeping FOSSBOX alive.
```

Comment:
i wonder if this could use more. it seems up cut off fairly abruptly with this sentence on what FOSS@RIT was. maybe we could give it a bit of a new name/purpose? especially with the reboot (ish) of FOSS-hours and my interest in promoting student projects with the discord server

Reply:
this may also make a good little mission statement for the "who we are" section of the homepage maybe